prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts

```diff
@@ -41,6 +43,11 @@ const routes: Routes = [
         component: SiteComponent,
         data: { title: 'Welcome to PRIME' }
       },
+      {
+        path: AuthRoutes.PHSA,
+        component: PhsaComponent,
+        data: { title: 'Enrol for access to PHSA eForms', subtitle: 'Request approval to access PHSA eForms', button: 'Logon with BCSC' }
```

Comment:
You won't want to pass in the button label through the route configuration that feels more like a direct template entry.  Likely the title is enough which will match the browser tab and page title.